warehouse/forklift/legacy.py

```diff
@@ -46,6 +50,17 @@
 
 PATH_HASHER = "blake2_256"
 
+ALL_STDLIB = set(
```

Comment:
Any reason for having ``ALL_STDLIB`` exist instead of just nesting this logic inside the comprehension for ``STDLIB_PROHIBITED``? We could ditch the "inner" ``set()`` then and just have ``STDLIB_PROHIBITED`` be a set comprehension.